rfd/0040-webauthn-support.md

```diff
@@ -40,35 +42,35 @@ to the server with the signed challenge. Assuming all is well this completes the
 registration process.
 
 ```
-                          Registration flow                     
```

Comment:
Note to reviewers: everything below this point is just automatic whitespace trimming.